package.json

```diff
@@ -28,7 +28,7 @@
   "peerDependencies": {
     "jest-cli": ">=18.0.0",
     "jest-runtime": ">=18.0.0",
-    "stryker-api": "^0.5.0"
+    "stryker-api": ">=0.5.0"
```

Comment:
Actually, `^0.5.0` means: at least version `0.5.0`, but no higher than excluding `0.6.0`, as `0.6.0` is the next major version according to [semver2](http://semver.org/spec/v2.0.0.html). This means that your not compatible with the latest stryker-api releases.

You could change it to be `"0.x.x"`. So we won't be automatically compatible with the "real first awesome major release", if that makes any sense.